README.md

```diff
@@ -1,2 +1,12 @@
 # VictiScout-Server
-Server for VictiScout.
+
+Server for VictiScout
+Setup:
+
+1. Create a virtualenv by doing ```virtualenv flask```
```

Comment:
If you're going to use virtualenvs, consider using virtualenvwrapper instead. Makes life easier.